cmd/cli/namespace_add.go

```diff
@@ -88,6 +90,14 @@ func (a *namespaceAddCmd) run() error {
 		ctx, cancel := context.WithCancel(context.Background())
 		defer cancel()
 
+		controllers, err := getControllerDeployementsForMesh(a.clientSet, a.meshName, "")
```

Comment:
I think it might be more succinct to have a `func meshExists(name, namespace) bool` and wrap the logic of controllers being present within the function.

Reply:
Addressed in latest changes.